libindy/indy-utils/src/crypto/base64/rust_base64.rs

```diff
@@ -1,4 +1,4 @@
-use errors::prelude::*;
+use indy_api_types::errors::prelude::*;
```

Comment:
I like moving errors to separate crat. We will avoid cross dependencies and duplication in `indy-api-types` and `indy-utils`

Reply:
duplication/cross dependencies fixed by moving next_handle_* helpers to utils